src/main/resources/build.template.properties

```diff
@@ -44,7 +44,7 @@ app.crashreport.email = app_admin@gmail.com
 # This is the url to display as a message of the day to a student.
 # Use test-student-motd.html to test this feature.
 # e.g. app.student.motd.url = test-student-motd.html
-app.student.motd.url = 
+app.student.motd.url = test-student-motd.html
```

Comment:
Note that you should change `build.properties` (which is not added to source control) and not `build.template.properties` (which is added to source control)